backend/local/changenotify_other.go

```diff
@@ -0,0 +1,275 @@
+//go:build !windows
```

Comment:
While there is some commonality between the implementation for Windows and other platforms, I've separated them out with conditional compilation to keep the logic clearer on reading.